assets/src/components/event-attendees/index.js

```diff
@@ -35,9 +36,10 @@ export default class EventAttendeeList extends Component {
 			return '';
 		}
 		const listItems = attendees.map(
-			( attendee, index ) => {
+			( attendee ) => {
+				const id = attendee.id || cuid();
```

Comment:
so this is safe from collisions, but will also generate a new cuid on every component rerender, which will in turn trigger a rerender for each item using a cuid for its key, since those items would now have a new cuid.

I wonder if there is a quick and easy way to either:

- generate a hash from some of the primary entity properties and use that (assuming those properties don't change)
- save a temp ID to each entity, such as a cuid, and then the entity id can be used and will stay with that entity for the entire time it exists in the store until it is persisted.

Reply:
> generate a new cuid on every component rerender

To be clear, a cuid is ONLY generated if there is no attendee id. I'm actually going to change this because `AttendeeListItem` EXPECTS an instance of `BaseEntity` so if there's no id it's going to bail anyways.  So what I'll do is just skip the list item if it doesn't have an id.